lib/unitsml.rb

```diff
@@ -36,3 +36,15 @@ def self.parse(string)
   dimension_quantity: Unitsml::Unitsdb::DimensionQuantity,
 }
 require "unitsdb"
+
+DEFAULT_XML_ADAPTER = if RUBY_ENGINE == "opal"
+  require "lutaml/model/xml_adapter/oga_adapter"
+  :oga
+else
+  require "lutaml/model/xml_adapter/ox_adapter"
+  :ox
+end
```

Comment:
```suggestion
DEFAULT_XML_ADAPTER = if RUBY_ENGINE == "opal"
  :oga
else
  :ox
end
```

I think the requires are not needed, this will be handled by the `lutaml-model`